app/models/event.rb

```diff
@@ -86,12 +86,53 @@ class Event < ApplicationRecord
 
   self.inheritance_column = nil
 
+  validates_with MergedCandidateIsReadOnlyValidator
   validates :type, presence: true
-  validates :eventable_type, presence: true
-  validates :eventable_id, presence: true # rubocop:disable Rails/RedundantPresenceValidationOnBelongsTo
 
   after_create :update_candidate_last_activity
 
+  def self.create_changed_event_if_value_changed(
+    eventable:,
+    changed_field: nil,
+    field_type: :singular,
+    performed_at: Time.zone.now,
```

Comment:
may we set `nil` ?

Reply:
idk, what will be better. Either way, it will default to Time.zone.now, by ruby's hands or postgres'

Reply:
exactly, no needed to set `now` if it will be set in postgres, this lint may confuse people that postres do not have default value